docs/elasticsearch/ux.md

```diff
@@ -0,0 +1,1081 @@
+---
+layout: default
+title: Search Experience
+parent: Elasticsearch
+nav_order: 12
+---
+
+# Search Experience
+
+Expectations from search engines have evolved over the years. Just returning relevant results quickly is no longer enough for most users. Elasticsearch includes many features that enhance the user’s search experience as follows:
+
+Feature | Description
+:--- | :---
+Autocomplete queries | Suggest phrases as the user types.
+Paginate results |  Rather than a single, long list, break search results into pages.
+Scroll search | Return a large number of results in batches.
+Sort results | Allow sorting results by different criteria.
+Highlight query matches | Highlight the search term in the results.
+
+---
+
+## Autocomplete queries
+
+Autocomplete shows suggestions to users while they type.
+
+For example, if a user types "pop," Elasticsearch provides suggestions like "popcorn" or "popsicles." These suggestions preempt your user's intention and lead them to a possible search term more quickly.
+
+Elasticsearch allows you to design autocomplete that’s:
```

Comment:
I'm not wild about the formatting here due to the sentence fragment, colon, and then list with colons. How do you feel about the following?

"design autocomplete that is responsive, relevant, and forgiving."

- Responsive: Update suggestions with each keystroke.
- Relevant: Rather than many low quality suggestions, provide only a few high quality ones.
- Forgiving: Tolerate typos.

Reply:
Would it be more clear to say "design autocomplete that updates with each keystroke, provides a few relevant suggestions, and tolerates typos."